clickhouse_driver/log.py

```diff
@@ -1,6 +1,30 @@
 import logging
 
-logger = logging.getLogger(__name__)
+
+def default_message_filter(record):
+    record.msg = (
+        f'[ {record.server_host_name} ] '
+        f'[ {record.server_event_time}.'
+        f'{record.server_event_time_microseconds:06d} ] '
+        f'[ {record.server_thread_id} ] '
+        f'{{{record.server_query_id}}} '
+        f'<{record.server_priority}> '
+        f'{record.server_source}: '
+        f'{record.server_text}'
+    )
+    return True
+
+
+def configure_logger(raw_log_record=False):
```

Comment:
BTW defaults can be error prone, and in this particular case it could be avoided, the function is not very common. So, I would rather remove the default here